Modules/_fuzz/_fuzzmodule.c

```diff
@@ -0,0 +1,52 @@
+#include <Python.h>
```

Comment:
Do we really need this module? It's only usage seems to be for the tests, and it looks like it has as much code as the fuzzers themselves!

Reply:
To me what I see in this code is pretty standard CPython API calls.  That raises the question of if there is a good reason to check these in and keep them working as part of CPython or if this code should be a stand alone module available on pypi that people could use to launch fuzz tests of any Python interpreter it is built with.  The answer to that question likely comes from answering how oss-fuzz the fuzz testing service works.  If it monitors our github repo and reruns fuzz tests as things change in cpython, including these here may make sense.  I think a comment describing the situation and why this exists here with links to relevant oss-fuzz documentation (either in here or in the _fuzz/README.rst) would help that.

Reply:
I've added a little to README.rst, but to summarize and expand here:

You're correct that oss-fuzz regularly pulls from CPython and runs the fuzz tests locally.

oss-fuzz prefers to have the tests be a tested part of the upstream project so that they don't break. Certainly not all of these fuzz tests are using the public API (e.g._Py_HashBytes), and there will be more in the future. Those are more likely to they might break in the future -- having them here ensures that breaking them would break the tests, and they will always be good to run, hopefully.

We could keep the public-API-using fuzz tests separate in oss-fuzz, but they wouldn't really like that, and that actually splits up the tests in a confusing way -- some would be here, some would be in the oss-fuzz repo. I think they're better off all being here.